x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/fixtures/page_objects/custom_logs.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ScoutPage, KibanaUrl, KbnClient } from '@kbn/scout';
+
+export class CustomLogsPage {
+  constructor(
+    private readonly page: ScoutPage,
+    private readonly kbnUrl: KibanaUrl,
+    private readonly kbnClient: KbnClient
+  ) {}
+
+  async goto() {
+    this.page.goto(`${this.kbnUrl.app('observabilityOnboarding')}/customLogs`);
+  }
+
+  async clickBackButton() {
+    await this.page.testSubj.click('observabilityOnboardingFlowBackToSelectionButton');
+  }
+
+  logFilePathList() {
+    return this.page.locator(`[data-test-subj^=obltOnboardingLogFilePath-]`);
+  }
```

Comment:
IMO, we shouldn't wrap the locators in functions which are not performing any action or are not parametrized. Keeping them as public instance variables would result in cleaner Page Objects, as shown in the [example in docs](https://playwright.dev/docs/pom).

@dmlemeshko what are your thoughts on this?

Reply:
That's a good point, I'll convert those without any parameters or actions into properties 👍

Reply:
Migrated to use properties